book/html.md

```diff
@@ -16,7 +16,18 @@ A tree of nodes
 ===============
 
 The HTML tree[^dom] has one node\index{node} for each open and close tag pair
-and for each span of text.[^1] So for our browser to use a tree, tokens need to
+and for each span of text.[^1]
+
+[^1]: In reality there are other types of nodes too, like comments,
+    doctypes, and `CDATA` sections, and processing instructions. There
+    are even some deprecated types!
+
+::: {.print-only .center}
+![An annotated HTML document](im/html-syntax.png)
+^[An HTML document, showing tags, text, and the nesting structure.]
```

Comment:
Why the footnote? Why not just rely on the alt text?

Reply:
Yeah, uhh, I don't know. This is how we do it in `history.md` and it looks terrible. I figured now wasn't the time to figure it out :(